contracts/src/v0.8/upkeeps/CronUpkeep.sol

```diff
@@ -103,26 +106,16 @@ contract CronUpkeep is KeeperCompatibleInterface, KeeperBase, ConfirmedOwner, Pa
    * @param id the id of the cron job to delete
    */
   function deleteCronJob(uint256 id) external onlyOwner {
```

Comment:
can we add a test that tests the revert?

![image](https://user-images.githubusercontent.com/14809513/146030666-c1199057-068d-4cc9-a3e0-5e1caf60dca9.png)

Reply:
also, can we move the if ... revert.. into a modifier? like `onlyValidCronID`  or something similar? we can use it here and in `getCronJob`

Reply:
done